lib/src/core/metadata/directives.dart

```diff
@@ -722,7 +727,7 @@ class DirectiveMetadata extends InjectableMetadata {
 ///
 /// ## Lifecycle hooks
 ///
-/// When the component class implements some [../../guide/lifecycle-hooks.html]
+/// When the component class implements some [lifecycle-hooks](docs/guide/lifecycle-hooks.html)
```

Comment:
OK, I've suggested wording at https://github.com/astashov/dartdocs.org/issues/30. Do you want me to provide the actual HTML?